src/core/blockchain.cr

```diff
@@ -386,11 +390,14 @@ module ::Sushi::Core
     end
 
     def headers
-      @chain.map { |block| block.to_header }
+      chain = @database.get_blocks(0_i64)
```

Comment:
I think in the long term calls like this will have to be paginated - so when we have 10 million blocks - we can by default return the 2 days worth from in memory - and then paginate to get more - similar to how transactions_for_address works